examples/with-jest-babel/package.json

```diff
@@ -4,7 +4,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "test": "jest --watch",
+    "test": "jest --watchAll",
```

Comment:
Is this a good way out though? See https://github.com/vercel/next.js/issues/69236#issuecomment-3687985527

Because:

> Watch files for changes and rerun all tests when something changes. If you want to re-run only the tests that depend on the changed files, use the --watch option.

In a large codebase this perhaps won't be so good? For this example specifically, it is a good defensive approach cuz, people might start the example with `--disable-git` though.